lib/src/delegates/assets_picker_text_delegate.dart

```diff
@@ -154,3 +154,54 @@ class EnglishTextDelegate implements AssetsPickerTextDelegate {
     return '$minute$separator$second';
   }
 }
+
+/// [AssetsPickerTextDelegate] implements with Hebrew.
+/// 希伯来文字实现
+class HebrewTextDelegate implements AssetsPickerTextDelegate {
+  factory HebrewTextDelegate() => _instance;
+
+  HebrewTextDelegate._internal();
+
+  static final HebrewTextDelegate _instance = HebrewTextDelegate._internal();
+
+  @override
+  String confirm = 'אישור';
+
+  @override
+  String cancel = 'ביטול';
+
+  @override
+  String edit = 'עריכה';
+
+  @override
+  String gifIndicator = 'GIF';
+
+  @override
+  String heicNotSupported = 'קובץ HEIC לא נתמך.';
+
+  @override
+  String loadFailed = 'הטעינה נכשלה';
+
+  @override
+  String original = 'מקור';
+
+  @override
+  String preview = 'תצוגה מקדימה';
+
+  @override
+  String select = 'בחר';
+
+  @override
+  String unSupportedAssetType = 'סוג קובץ אינו נתמך';
+
+  @override
+  String durationIndicatorBuilder(Duration duration) {
```

Comment:
Sorry for not seeing this, but what do you think if we implement this as a default implementation in the abstract delegate?

Reply:
After that we don't have to override this method in most of delegates.

Reply:
Correct! This will be much better. I'm on it.

Reply:
Apparently, this is not trivial in dart. Dart forces implementation of non abstract methods in child classes, and if I change from "implements" to "extends" I'm unable to override the String fields.
https://stackoverflow.com/questions/61738325/dart-forcing-me-to-implement-non-abstract-method-in-child-class
What do you think?

Reply:
Maybe:
![image](https://user-images.githubusercontent.com/15884415/116509081-41d71280-a8f5-11eb-9662-df4081a3bad8.png)
![image](https://user-images.githubusercontent.com/15884415/116509095-47ccf380-a8f5-11eb-8f4a-956da58df88b.png)

Reply:
Coo, static solves this issue. Implementing.

Reply:
See changes, I have also commited a fix to the Hebrew translation for unSupportedAssetType (as per the English change revert).